labs-ios-starter/Application/Extensions/String + identifier.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  String + identifier.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/12/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+enum StoryboardIdentifier: String {
+    case profileDetailViewController
+    case surveyTabBarController
+    case profileNavigationController
+}
+
+enum SegueIdentifier: String {
+    case showProfileDetail
+    case showDetailProfileList
+    case modalAddProfile
+}
+
+enum TableViewIdentifier: String {
+    case profileCell
+}
+
+extension String {
+    ///convenience method to retrieve Storyboard Identifiers
+    static func getStoryboardIdentifier(identifier: StoryboardIdentifier) -> String {
+        //storyboard ids are capitalized
+        identifier.rawValue.firstCapitalized
```

Comment:
Consider having the cases themselves be capitalized, or their raw values be capitalized, and then calling the enum directly. Or if you wanted this to be called like a string, maybe you could extend string itself to include the StoryboardIdentifier enum. Would be nice to access this value with built in getter, unless I’m missing something here.